rest_framework/fields.py

```diff
@@ -1456,7 +1457,7 @@ class MultipleChoiceField(ChoiceField):
         'not_a_list': _('Expected a list of items but got type "{input_type}".'),
         'empty': _('This selection may not be empty.')
     }
-    default_empty_html = []
+    default_empty_html = []  # type: List[Any]
```

Comment:
OK, so can'y `mypy` just infer this? Surely all lists are `List[Any]`, like the `= []` already implies that no?

Reply:
It looks like mypy can't infer the type of empty lists, so that may be why it was specified here.

https://mypy.readthedocs.io/en/latest/type_inference_and_annotations.html#explicit-types-for-collections

Reply:
To clarify, it looks like MyPy follows the "Explicit is better than implicit" portion of the Zen of Python and doesn't assume `List[Any]` when it's not specified and the list is empty.

Reply:
Sure. 

(Grumble, grumble... elsewhere lack of an annotation is interpreted as `Any`... 🙂)